### PR TITLE
Now Fsa.as_dict() saves tensor attributes as well as non-tensor attributes

### PR DESCRIPTION
**Pull request by csukuangfj**
Ready for review.

### CODE REVIEW

File: k2/python/k2/fsa.py [hunk 1]
```diff
@@ -193,25 +193,34 @@ def __init__(self,
...
-        ans += "properties_str = " + _k2.fsa_properties_as_str(
-            self._properties) + "."
+        ans += 'properties_str = ' + _k2.fsa_properties_as_str(
+            self._properties) + '.'
+        for name, value in self.named_tensor_attr(include_scores=False):
+            sep = '\n'
+            ans += f'{sep}{name}: {value}'
+        for name, value in self.named_non_tensor_attr():
+            sep = '\n'
+            ans += f'{sep}{name}: {value}'
```
Comment (danpovey) [inline k2/python/k2/fsa.py:222]: mm I'm concerned if we us symbol tables a lot, this could be too verbose.

**Comment by danpovey** [comment]
Looks OK to me.

File: k2/python/k2/fsa.py (hunk 1 above)
Comment (csukuangfj) [reply k2/python/k2/fsa.py:222]: Ok, I will remove the symbol table from the output.

**Comment by csukuangfj** [comment]
Merging.

**Pull request merged**